packages/ocean-react/src/Alert/Alert.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import AlertIcon from './AlertIcon';
+
+export type AlertProps = {
+  /**
+   * Determines the type of alert, with default icon and colors for each type
+   * @default 'default'
+   */
+  type?: 'success' | 'warning' | 'error' | 'default';
+  /**
+   * Sets a title for the Alert
+   * @default false
+   */
+  title?: string | false;
+  /**
+   * Sets a title for the Alert
+   * @default false
+   */
+  icon?: React.ReactNode | false;
+} & React.ComponentPropsWithoutRef<'div'>;
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  {
+    children,
+    type = 'default',
+    title = false,
+    className,
+    icon = false,
+    ...rest
+  },
+  ref
+) {
+  return (
+    <div
+      {...rest}
```

Comment:
It's most common in our library to put the `{...rest}` at the end, but if you want to leave so no problem.